tests.py

```diff
@@ -605,3 +607,35 @@ def read_data(expected):
         self.assertEqual(light.brightness, 80)
         self.assertEqual(light.getRgb(), (1, 25, 80))
         self.assertEqual(light.device_type, flux_led.DeviceType.Bulb)
+
+    def test_rgbww_brightness(self):
```

Comment:
I attempted to extract the code from https://github.com/icemanch/core/blob/Flux_LED-ColorSupport/homeassistant/components/flux_led/light.py#L543

Please double check that I didn't make a mistake splitting it